test/core_tests/config_test.py

```diff
@@ -376,6 +376,18 @@ def test_config_sensor_percent(self):
                    'Heating.Valve1',
                    group_address_state='2/0/0',
                    value_type='percent',
+                   sync_state=True,
```

Comment:
Isnt this the default value? (feel free to keep or remove it)

Reply:
Yes it is.